plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -219,12 +219,15 @@ public Version getVersion() throws IOException {
      * @throws InterruptedException
      *         if build process was interrupted
      */
-    public String buildImage(String repository, ProgressMonitor progressMonitor, AuthConfigs authConfigs, File... files)
-            throws IOException, InterruptedException {
+    public String buildImage(String repository,
+                             ProgressMonitor progressMonitor,
+                             AuthConfigs authConfigs,
+                             boolean doForcePull,
```

Comment:
maybe if new params will be added middle-term, to introduce an object parameter containing the options.
Thus, signature won't be changed on many places each time a new param is added